src/Common/src/System/StringNormalizationExtensions.Unix.cs

```diff
@@ -10,34 +10,34 @@ namespace System
     static partial class StringNormalizationExtensions
     {
         [SecurityCritical]
-        public static bool IsNormalized(this string value, NormalizationForm normalizationForm)
+        public static bool IsNormalized(this string strInput, NormalizationForm normalizationForm)
```

Comment:
Never mind I think it is because that is what they are called on desktop and you want to make them consistent?